36/javadoc/allclasses-index.html

```diff
@@ -11,7 +11,7 @@
 <link rel="stylesheet" type="text/css" href="script-dir/jquery-ui.min.css" title="Style">
 <link rel="stylesheet" type="text/css" href="jquery-ui.overrides.css" title="Style">
 <script type="text/javascript" src="script.js"></script>
-<script type="text/javascript" src="script-dir/jquery-3.6.1.min.js"></script>
```

Comment:
why the downgrade? How were the `3.6.0` javadoc files generated?

Reply:
It's because the previous javadoc files were generated using a different JVM. I used `OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)`, while the previous version seems to have used (from the copyright/license files) an Oracle JVM.

Reply:
Makes sense